components/multiselect/multiselect.tsx

```diff
@@ -360,76 +493,55 @@ export default class MultiSelect<T extends Value> extends React.PureComponent<Pr
-                        />
-                    </div>
-                    <div
-                        id='multiSelectHelpMemberInfo'
-                        className='multi-select__help'
-                    >
-                        {numRemainingText}
-                        {memberCount}
+            <React.Fragment>
+                <div className='filtered-user-list'>
+                    <div className='filter-row filter-row--full'>
+                        <div className='multi-select__container react-select'>
+                            {reactSelectDiv}
+                            {this.props.saveButtonPosition === 'top' &&
+                            <SaveButton
+                                id='saveItems'
+                                extraClasses='ml-4'
+                                saving={this.props.saving}
+                                disabled={this.props.saving}
+                                onClick={this.handleOnClick}
+                                defaultMessage={buttonSubmitText}
+                                savingMessage={this.props.buttonSubmitLoadingText}
+                            />}
+                        </div>
+                        <div
+                            id='multiSelectHelpMemberInfo'
+                            className='multi-select__help'
+                        >
+                            {numRemainingText}
+                            {memberCount}
+                        </div>
                     </div>
+                    {multiSelectList}
                     <div
                         id='multiSelectMessageNote'
                         className='multi-select__help'
                     >
                         {noteTextContainer}
                     </div>
+                    {this.props.saveButtonPosition === 'top' &&
+                    <div className='filter-controls'>
+                        {previousButton}
+                        {nextButton}
+                    </div>}
                 </div>
-                <MultiSelectList
-                    ref={this.listRef}
-                    options={optionsToDisplay}
-                    optionRenderer={this.props.optionRenderer}
-                    ariaLabelRenderer={this.props.ariaLabelRenderer}
-                    page={this.state.page}
-                    perPage={this.props.perPage}
-                    onPageChange={this.props.handlePageChange}
-                    onAdd={this.onAdd}
-                    onSelect={this.onSelect}
-                    loading={this.props.loading}
-                    selectedItemRef={this.props.selectedItemRef}
-                />
-                <div className='filter-controls'>
-                    {previousButton}
-                    {nextButton}
-                </div>
-            </div>
+                {this.props.saveButtonPosition === 'bottom' &&
+                <div className='text-right pr-6 pb-6'>
```

Comment:
nit: more semantic classes

Reply:
^ this one too

Reply:
Are we sure we want to do that, we would then have to create css rules for every simple thing we want to do if we want to be consistent I guess. So if I have to just give it a margin-left of 4px, I would have to create a rule for that in every separate instance instead of reusing a helper class right?